src/styles/components/overrides/_extension-cards.scss

```
@@ -198,7 +198,7 @@
 
   &-footer {
     @include flex-between;
-    border-top: 1px solid var(--border-light);
+    background: var(--modal-footer);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)